Packs/MicrosoftIntune/README.md

```diff
@@ -0,0 +1,24 @@
+# Microsoft Intune
+
+This pack includes Cortex XSIAM content.
+
+Note: The logs will be stored in the dataset named *msft_azure_raw*. 
+To filter a query to focus only on Microsoft Intune logs, use the following filters:
+- In XQL queries, use: *|filter _collector_name=<Name_of_Instance_from_Azure_Event_Hub_Configuration>*
+- In Datamodel queries, use: *|xdm.observer.name=<Name_of_Instance_from_Azure_Event_Hub_Configuration>*
+
+## Collect Events from Vendor
+
+In order to use the collector, you need to use the following option:
+
+- [Microsoft Intune](#microsoft-intune)
```

Comment:
I think these links should be removed.  The second link, links to the section it is in.  This Readme is not so large that we need to have these links.